nemo/collections/common/tokenizers/text_to_speech/tts_tokenizers.py

```diff
@@ -238,6 +238,71 @@ def __init__(
         )
 
 
+class RussianCharsTokenizer(BaseCharsTokenizer):
+    def __init__(
+        self,
+        punct=True,
+        apostrophe=True,
+        add_blank_at=None,
+        pad_with_space=False,
+        non_default_punct_list=None,
+        text_preprocessing_func=any_locale_text_preprocessing,
+    ):
+        """Russian phoneme-based tokenizer.
```

Comment:
Could you change this description to be more accurate?

Input text is assumed to be a mixture of IPA symbols (and cyrillic characters for words not handled by G2P).

Note that neither are phonemes. Russian has 5 to 6 vowel phonemes (depending on who you ask). Here I see a lot of tokens for vowels (əɪɐɨaʊioueʉæɵɛ).